apps/cli/commands/extract_status.py

```
@@ -0,0 +1,189 @@
+        _display_queue_depth(status)
+        _display_metrics(status)
+
+        logger.info("Status display complete")
+
+    except typer.Exit:
+        # Re-raise typer.Exit to preserve exit code
+        raise
+    except Exception as e:
+        # Catch any other errors and report them
+        console.print(f"[red]✗ Failed to retrieve system status: {e}[/red]")
+        logger.error(f"Status collection failed: {e}", exc_info=True)
```

Ruff (0.14.1)

Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)